src/Keyboard/Button.php

```diff
@@ -12,17 +14,17 @@ class Button
 
     private int $width = 0;
 
-    private function __construct(
+    final public function __construct(
```

Comment:
may I ask why are you changing this one's visibility?

Reply:
I don't know, maybe I was drunk